aether-kernel/aether/kernel/api/tests/test_serializers.py

```diff
@@ -344,8 +345,12 @@ def test__serializers__create_and_update(self):
             context={'request': self.request},
         )
         self.assertTrue(entity_6.is_valid(), entity_6.errors)
-        entity_6.save()
-        self.assertEqual(entity_6.data['project'], mapping.data['project'])
+        with self.assertRaises(ValidationError) as ve:
+            entity_6.save()
+        self.assertIn(
```

Comment:
Indent the block inside the with statement.
```python
with self.assertRaises(ValidationError) as ve:
    entity_6.save()
    self.assertIn(
        'Schema Decorator MUST be provided with entities',
        str(ve.exception)
    )
```

Reply:
still pending ;-)

Reply:
:sob: